README.md

```diff
@@ -92,3 +92,27 @@ Best practice for managing a pull request
 
      ```make gh-publish```
 
+
+Use docker to build and preview the website
+--------------------------------------------
+To build CNERG website using docker you can use the cnerg container containing
+all the required dependencies to build the website cnerg/cenrg_XXXX.
+
+1. go into your folder hosting cnerg website sources
+
+2. run the docker container: 
+
+    ```docker run -v $PWD:/local_drive -it baaaaam/cnerg```
+
+3. go into the mounted folder in docker: 
+    ```cd /local_drive```
+
+4. build a preview of the website: 
+    ```make gh-preview```
+
+5. quit docker: 
+    ```exit```
+
+6. preview your modify version of the website: 
+
+    ```open gh-build/index.htlm````
```

Comment:
That command is Mac-specific I think. I'd just indicate that they can open the `index.html` file from their browser at that point. Would be nice to indicate that the file will be in the `gh-build` folder as well.

Reply:
hltm -> html